Domains/2-DisciplinePhysical/StructuralPhysical/StructuralPhysical.ecschema.xml

```diff
@@ -25,10 +25,10 @@
             </Deprecated>
         </ECCustomAttributes>
     </ECEntityClass>
-    <ECEntityClass typeName="StructuralElement" displayLabel="Structural Element" modifier="Abstract">
+    <ECEntityClass typeName="StructuralElement" displayLabel="Structural Element" modifier="Abstract" description="A bis:PhysicalElement is a general abstract class for structural items.">
         <BaseClass>bis:PhysicalElement</BaseClass>
     </ECEntityClass>
-    <ECEntityClass typeName="StructuralMember" displayLabel="Structural Member">
+    <ECEntityClass typeName="StructuralMember" displayLabel="Structural Member" description="A bis:StructuralElement; is the base class of all structural items.">
```

Comment:
Both descriptions for StructuralElement and StructuralMember talk about "being the base class for structural items". It is confusing. Feel free to ask help from @jchick-bentley in order to come up with clearer single-sentence descriptions about these two classes.

Reply:
The main difference I found between them, if I understood correctly, is that StructuralElement is an abstract class, meaning no instances of this class could be created. On the other hand, StructuralMember is a base class that can be instantiated directly or used to define another item that is not yet fully specialized in the schema, like a Cable, for example. Is this correct?

Reply:
@jchick-bentley can confirm but I think your understanding is correct. Maybe the usage of the wording "base class" in the description of "StructuralMember" is what gets confusing, since such wording typically gives the idea of an "abstract class". But it is true that such class being "intermediate" in the hierarchy and "concrete/non-abstract" at the same time is what makes it confusing.

How about:

StructuralElement: "A bis:PhysicalElement used to provide Structural support."
StructuralMember: "A generically-classified StructuralElement, unless a subclass is used for further classification"

BTW, I just noticed the usage of the bis: prefix with classes defined in the StructuralPhysical schema. We have been trying to use the "alias of a referenced schema" (e.g. bis: for BisCore) when talking about a concept defined outside of the current schema. That is, bis:PhysicalElement vs. just StructuralElement since that class is defined in the same StructuralPhysical schema.